docs/produce-events.md

```diff
@@ -0,0 +1,267 @@
+    import org.apache.kafka.clients.producer.ProducerConfig;
+    import org.apache.kafka.clients.producer.ProducerRecord;
+    import org.apache.kafka.common.config.SaslConfigs;
+    import org.apache.kafka.common.config.SslConfigs;
+    import org.apache.kafka.common.errors.WakeupException;
+    import org.slf4j.Logger;
+    import org.slf4j.LoggerFactory;
+
+    import io.confluent.kafka.serializers.AbstractKafkaAvroSerDeConfig;
+    import io.confluent.kafka.serializers.KafkaAvroSerializer;
+    import io.confluent.kafka.serializers.subject.TopicRecordNameStrategy;
+
+    import java.util.Properties;
+    import java.util.concurrent.atomic.AtomicBoolean;
+
+    public class TestProducer {
+      private static final Logger LOG = LoggerFactory.getLogger(TestProducer.class);
+
+      // Producer values
+      private static final String TOPIC = "test";
+      private static final String EB_BOOTSTRAP_SERVERS = System.getenv("EB_BOOTSTRAP_SERVERS");
+      private static final String EB_SECURITY_PROTOCOL = System.getenv("EB_SECURITY_PROTOCOL");
+      private static final String SCHEMA_REGISTRY_URL = System.getenv("SCHEMA_REGISTRY_URL");
+
+      private final KafkaProducer<Long, GenericRecord> producer;
+
+      public TestProducer() {
+        this.producer = createProducer();
+      }
+
+      public void run() {
+        try {
+          // Configure properties for the KafkaAvroSerializer
+          Properties serializerProps = new Properties();
+          serializerProps.put(AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, SCHEMA_REGISTRY_URL);
+          serializerProps.put(AbstractKafkaAvroSerDeConfig.VALUE_SUBJECT_NAME_STRATEGY,
+              TopicRecordNameStrategy.class.getName());
+
+          // Create KafkaAvroSerializer
+          KafkaAvroSerializer avroSerializer = new KafkaAvroSerializer();
+          avroSerializer.configure(serializerProps, false);
+
+          while (true) {
+            // Retrieve the Avro schema from the Schema Registry, e.g. the schema named
+            // "test-value"
+            Schema schema = avroSerializer.getSchemaRegistry().getSchemaBySubjectAndVersion(TOPIC + "-value",
+                avroSerializer.getVersion(TOPIC + "-value"));
+
+            // Create a new Avro record
+            GenericRecord record = new GenericData.Record(schema);
+            record.put("name", "John Doe");
+            record.put("email", "john.doe@example.com");
+
+            // Create producer record
+            ProducerRecord<Long, GenericRecord> producerRecord = new ProducerRecord<>(TOPIC, record);
+
+            // Send the record to the Kafka topic
+            producer.send(producerRecord);
+          }
+        } catch (final Exception e) {
+          LOG.error("An exception occurred while producing messages", e);
+        } finally {
+          producer.close();
+        }
+      }
+
+      public void shutdown() {
+        shutdown.set(true);
+      }
+
```

Comment:
```suggestion
```

This and the AtomicBoolean import can be removed

Reply:
Thanks! I've made the changes :)